org.eclipse.iot.fourdiac.Ide.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "org.eclipse.iot.fourdiac.Ide",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
```

Comment:
which bits of GNOME does this want that aren't in the FDo runtime (which certainly has GTK already for instance)?